File: 1446-consecutive-characters/1446-consecutive-characters.cpp
class Solution {
public:
    int maxPower(string s) {
        int j=0;
        int n=s.size();
        int count=1;
        int maxlength=1;
        for(int i=1;i<n;i++){
            if(s[i]==s[i-1]){
                count++;
            }else{
                count=1;
            }
            maxlength=max(count,maxlength);

        }
        return maxlength;
    }
};

File: 1446-consecutive-characters/1446-consecutive-characters.java
class Solution {
    public int maxPower(String s) {
        int n=s.length();
        int length=1;
        int maxlength=1;
        for(int i=0;i<n-1;i++){
            if(s.charAt(i)==s.charAt(i+1)){
                length++;
            }else{
                length=1;
            }

            maxlength=Math.max(length,maxlength);
        }
        return maxlength;
    }
}